### PR TITLE
Document rough consensus on reverting changes

### PR DESCRIPTION
**Pull request by foolip**
Fixes https://github.com/web-platform-tests/wpt/issues/9953.

### CODE REVIEW

**Comment by foolip** [comment]
@web-platform-tests/admins please review.

File: docs/_appendix/reverting.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    master branch or on pull requests following the breaking change.
+
+  * Breakage in browser engine repositories which import and run
+    web-platform-tests, such as Chromium, Edge, Gecko, Servo and WebKit.
+
+  * Breakage in results collections systems for results dashboards, such as
+    [wpt.fyi](https://wpt.fyi).
+
+  * Breakage in supplemental tooling used by working groups, such as the
+    [CSS build system][].
```
Comment (zcorpan) [inline docs/_appendix/reverting.md]: url?

File: docs/_appendix/reverting.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+When such breakage happens, if the maintainers of the affected systems request
+it, pull requests to revert the original change should normally be approved and
+merged as soon as possible. (When the origin change itself was fixing a serious
+problem, it's a judgement call, but prefer the fastest path to a stable state
+acceptable to everyone.)
+
+Once a revert has happened, the maintainers of the affected systems are
+expected to work with the original patch author to resolve the problem so that
+the change can be relanded. A reasonable timeframe to do s is within one week.
```
Comment (zcorpan) [inline docs/_appendix/reverting.md]: s -> so

File: docs/_appendix/reverting.md (hunk 1 above)
Comment (zcorpan) [reply docs/_appendix/reverting.md]: oh sorry, I don't know markdown syntax it seems.

Comment (foolip) [reply docs/_appendix/reverting.md]: This isn't even Markdown I'm afraid, it's Jekyll. Quite easy to get things wrong :)

**Review by jgraham — APPROVED** [review]

**Review by zcorpan — APPROVED** [review]

**Review by plehegar — APPROVED** [review]

File: docs/_appendix/reverting.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  * Breakage in results collections systems for results dashboards, such as
+    [wpt.fyi](https://wpt.fyi).
+
+  * Breakage in supplemental tooling used by working groups, such as the
+    [CSS build system][].
+
+When such breakage happens, if the maintainers of the affected systems request
+it, pull requests to revert the original change should normally be approved and
+merged as soon as possible. (When the origin change itself was fixing a serious
```
Comment (jdm) [inline docs/_appendix/reverting.md]: s/origin/original/

**Review by jdm — APPROVED** [review]

**Review by Ms2ger — APPROVED** [review]

**Review by gsnedders — APPROVED** [review]

**Pull request merged**